### PR TITLE
Added login page after logging thorugh socials

### PR DESCRIPTION
**Pull request by ALIRAZA17**
Added login page after logging thorugh socials

Refers to this issue #88

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,21 +0,0 @@
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
-
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
-DATABASE_URL=postgresql://admin:admin@localhost:5432/next_app_localhost_db?sslmode=verify-full
-NEXTAUTH_URL=http://localhost:3000/api/auth
-DISCORD_CLIENT_ID=somesecret
```
Comment (RobinNagpal) [inline .env.example:8]: Why did we delete this?

File: package.json [hunk 2]
```diff
@@ -108,7 +108,7 @@
     "zlib": "^1.0.5"
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "^3.3.1",
+    "@graphql-codegen/cli": "^5.0.0",
```
Comment (RobinNagpal) [inline package.json:111]: Why do we need this change?

File: .env.example (hunk 1 above)
Comment (ALIRAZA17) [reply .env.example:8]: I changed it to .env direclty let me fix it

File: package.json (hunk 2 above)
Comment (ALIRAZA17) [reply package.json:111]: We dont need this we can discard it its just that I had to clone the code again but we can discard it

File: src/app/api/auth/update-user/route.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from 'next/server';
+import { prisma } from '@/prisma';
+
+export async function PUT(request: Request) {
+  const { name, email, phone_number, username, spaceId } = await request.json();
+  try {
```
Comment (RobinNagpal) [inline src/app/api/auth/update-user/route.ts:6]: Any one can call the API and pass a username and spaceId.

Like anyone if they your username and spaceId, will be able to update your information

Comment (ALIRAZA17) [reply src/app/api/auth/update-user/route.ts:16]: Added Protection!!

File: src/components/main/TopNav/TopNav.tsx [hunk 4]
```diff
@@ -184,6 +198,9 @@ export default function TopNav(props: { space?: SpaceWithIntegrationsFragment |
                   {space && <DesktopNavLinks space={space} />}
                 </div>
                 <div className="flex items-center">
+                  <div className="mr-2">
```
[the remark this answers is not in the document]
Comment (ALIRAZA17) [reply src/components/main/TopNav/TopNav.tsx:201]: ok!

**Pull request merged**